readthedocsext/theme/templates/projects/partials/project_create_automatic.html

```diff
@@ -99,15 +100,26 @@
 
     </div>
     <div class="ui small bottom attached center aligned message">
+      <p>{% trans "Can't find the repository you are searching for?" %}</p>
 
-      {% trans "Can't find the repository you are searching for?" %}
+      <p>
+        {% if user|user_has_github_app_account %}
+          <a class="ui mini black basic compact button"
+             href="https://github.com/apps/{{ GITHUB_APP_NAME }}/installations/new/"
+             target="_blank">
+            <i class="fa-brands fa-github icon"></i>
+            {% trans "Install GitHub App" %}
```

Comment:
```suggestion
            {% trans "Install GitHub App (beta)" %}
```

Reply:
We should be consistent with how we are describing this, we don't really describe this as a beta anywhere else except maybe the migration page that blocks the last few steps.

I'm find labeling it as beta, but we should do it everywhere.

Reply:
Regarding content, this doesn't communicate that this step might be required every single project creation. If I had already connected/installed the GHA, I would assume I didn't need to click this button ever again.

It should specifically call out that the user needs to update the GHA permissions.

Reply:
I think this is the only real thing stopping this PR from going out. We can make more improvements in a following UI.

```suggestion
            {% trans "Update GitHub App permissions" %}
```